src/index.ts

```diff
@@ -485,6 +487,142 @@ export function apply(ctx: Context, config: Config) {
       }
     })
 
+  const cmdLsEx = ctx.command('lsmd <modelName:text>')
```

Comment:
The command should be `novelai.listmodel` at first, and then add `lsmd` as an alias.